site/docs/pages/guides/build-onchain-apps.mdx

```diff
@@ -31,8 +31,6 @@ Play with it live [here](https://onchain-app-template.vercel.app).
 
 ## Getting Started
 
-::::steps
-
```

Comment:
is this intentional?

Reply:
![Uploading Screenshot 2024-12-18 at 9.28.20 AM.png…]()

Reply:
<img width="493" alt="Screenshot 2024-12-18 at 9 28 39 AM" src="https://github.com/user-attachments/assets/711a3ee1-d900-428c-94aa-68f5a3c715a7" />

Reply:
that's odd, it should correspond with :::: at line 92, shows correctly currently https://onchainkit.xyz/guides/build-onchain-apps